src/test/shell/bazel/bazel_symlink_test.sh

```diff
@@ -70,9 +70,25 @@ function set_up() {
   mkdir -p symlink
   touch symlink/BUILD
   cat > symlink/symlink.bzl <<EOF
+def _dangling_symlink_impl(ctx):
+    symlink = ctx.actions.declare_symlink(ctx.label.name)
+    ctx.actions.symlink(
+        output = symlink,
+        target_path = ctx.attr.link_target,
+    )
+    return DefaultInfo(files = depset([symlink]))
+
+dangling_symlink = rule(
+    implementation = _dangling_symlink_impl,
+    attrs = {"link_target": attr.string()},
+)
```

Comment:
I think this whole definition is identical to the `symlink` rule below. Delete and just test the dangling behavior with the `symlink` rule?